src/tools/rbd_mirror/ImageReplayer.cc

```diff
@@ -379,6 +377,14 @@ void ImageReplayer<I>::handle_bootstrap(int r) {
     m_bootstrap_request = nullptr;
   }
 
+  // set m_delete_requested early to ensure that in case remote
+  // image no longer exists local image gets deleted even if start
+  // is canceled
```

Comment:
For consistency with `on_start_interrupted()` name:
```suggestion
  // is interrupted
```

Reply:
Done.